Library/Homebrew/cask/dsl.rb

```diff
@@ -259,6 +337,14 @@ def sha256(arg = nil, arm: nil, intel: nil)
       end
     end
 
+    # Sets the cask's archtecture strings.
```

Comment:
```suggestion
    # Sets the cask's architecture strings.
```

Not for this PR but: if there's any way to check these comments for typos like this: that'd be good.

Maybe some of the `docs` CI jobs for the generated YARD output?

Reply:
That would be awesome.